MapboxNavigation/RouteTableViewController.swift

```diff
@@ -68,20 +68,25 @@ class RouteTableViewController: UIViewController {
 extension RouteTableViewController: UITableViewDelegate, UITableViewDataSource {
 
     func numberOfSections(in tableView: UITableView) -> Int {
-        return 1
+        return routeController.routeProgress.route.legs.count
+    }
+    
+    func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        return "Leg \(section + 1)"
```

Comment:
The leg number is unimportant to the user. Instead, we should title the section with either of the following formats, preferably the first one:

* “Biloxi to Mobile” (see `leg.source`, `leg.destination`)
* “I-10” (see `leg.name`)